API v2 (#27)
Patreon/patreon-ruby: README.md

# patreon-ruby
Interact with the Patreon API via OAuth.

## Important notice about updating to 1.0.0 from earlier versions

> Patreon Ruby library version 1.0.0 moves on to Patreon's v2 API, which is not compatible with old v1 calls. Therefore directly upgrading from older versions to 1.0.0 would break compatibility of your installation. APIv1 will be deprecated some time soon after APIv2 come out of beta, so its important to get your integration compatible with API v2. 

Reviewer: ```suggestion
Patreon Ruby library version 1.0.0 moves on to Patreon's v2 API, which is not compatible with old v1 calls. Therefore directly upgrading from older versions to 1.0.0 would break compatibility of your installation. APIv1 will be deprecated some time soon after APIv2 comes out of beta, so it's important to get your integration compatible with API v2. 
```

> For more information on the differences/similarities between v1 and v2 please check: https://docs.patreon.com/#what-39-s-new

Reviewer: ```suggestion
For more information on the differences and similarities between v1 and v2, check out: https://docs.patreon.com/#what-39-s-new
```

* `get_webhooks(opts = { includes: [], fields: [],  count: 10, cursor: nil})`
* `get_campaigns_by_id_members(resource_id, opts = { includes: [], fields: [],  count: 10, cursor: nil})`
* `get_campaigns_by_id(resource_id, opts = { includes: [], fields: [], })`
* `get_webhooks_by_id(resource_id, opts = { includes: [], fields: [], })`
* `get_members_by_id(resource_id, opts = { includes: [], fields: [], })`

The `includes` and `fields` arguments to these methods specify
the [related resources](http://jsonapi.org/format/#fetching-includes)
and the [resource attributes](http://jsonapi.org/format/#fetching-sparse-fieldsets)
you want returned by our API, as per the [JSON:API specification](http://jsonapi.org/).
The lists of valid `includes` and `fields` arguments are provided on `Patreon::Schemas`.

For instance, if you wanted to request the first name and total amount a patron has ever paid to your campaign, you could do:
```ruby
api_client = Patreon::API.new(patron_access_token)

User = Patreon::Schemas::User
Member = Patreon::Schemas::Member

patron_response = api_client.get_identity({
    'includes': [User::Relationships::MEMBERSHIPS],

>     'fields': {

Reviewer: ```suggestion
    includes: [User::Relationships::MEMBERSHIPS],
    fields: {
```

Inside a hash literal, `k: e` is a shorthand for `:k => e`, so unless the intent is to construct a symbol with spaces in it, there is no need to quote the key.
- Not necessary: `'foo': e`, prefer: `foo: e`
- Okay: `'foo bar': e`, same as: `:'foo bar' => e`

Author: Good to know!
